packages/api/src/module/write/email-sender/infrastructure/node-mailer-email-sender.ts

```diff
@@ -0,0 +1,28 @@
+import {EmailSender} from "@/write/email-sender/application/email-sender";
+import Mail from "nodemailer/lib/mailer";
+import NodeMailer from 'nodemailer';
+
+export class NodeMailerEmailSender implements EmailSender {
+  private readonly transporter: Mail;
+  readonly mailFrom: string;
```

Comment:
I'm not sure if mailFrom should be here. More flexible solution may be to pass it as argument of sendAnEmail method (let's say we may want to send emails from mail like `rekrutacja@coderscamp.pl` / `mentorzy@coderscamp.pl` etc.) and use env variable in CommandHandler. 
What do you think? 
@KonradSzwarc - your opinion may be useful here.